packages/vaadin-themable-mixin/vaadin-themable-mixin.js

```diff
@@ -1,83 +1,229 @@
-import { DomModule } from '@polymer/polymer/lib/elements/dom-module.js';
+import { CSSResult, css, unsafeCSS } from 'lit';
```

Comment:
Please add license comment to this file.

Reply:
Done.